repository/src/lib.rs

```diff
@@ -196,8 +196,88 @@ impl<T: RawRepository> DistributedRepository<T> {
     ///
     /// Note that the proof in the `fp` branch must be set for the candidate commit
     /// for the last finalized block, which is `block_commit`.
-    pub async fn sync(&mut self, _block_commit: &CommitHash) -> Result<(), Error> {
-        unimplemented!()
+    pub async fn sync(&mut self, block_commit: &CommitHash) -> Result<(), Error> {
```

Comment:
```suggestion
    pub async fn sync(&mut self, block_commit_hash: &CommitHash) -> Result<(), Error> {
```
rename all please

Reply:
see this comment below
https://github.com/postech-dao/simperby/pull/189/files#r1039150915

for naming consistently in this implementation, we need to distinguish commit types and commit hash types

Reply:
fixed.